src/mkdocstrings_handlers/python/templates/material/_base/function.html

```diff
@@ -13,49 +13,51 @@
     {% set show_full_path = config.show_object_full_path %}
   {% endif %}
 
-  {% if not root or config.show_root_heading %}
+  {%- if config.show_function_title -%}
+    {% if not root or config.show_root_heading %}
 
-    {% filter heading(heading_level,
-        role="function",
-        id=html_id,
-        class="doc doc-heading",
-        toc_label=function.name ~ "()") %}
+      {% filter heading(heading_level,
+          role="function",
+          id=html_id,
+          class="doc doc-heading",
+          toc_label=function.name ~ "()") %}
 
```

Comment:
**For review purposes:**

the entire diff in this `function.html` file is just me adding a conditional with `config.show_function_title` and indenting everything within it.

Reply:
Noted :slightly_smiling_face: